assignment/scripts/aboutMe.js

```diff
@@ -1,51 +1,77 @@
 // REQUIRED FEATURES:
 // 1 - Create a variable called `firstName` and assign it the value of your first name
-
+let firstName = 'Tyler';
 // 2 - Create a second variable called `lastName` and assign it the value of your last name
-
+let lastName = 'Lowe';
 // 3 - Create a third variable called `fullName`, assign it the value of your first and last name
 // (remember, you already have variables for this, can you use those?)
+let fullName = `${firstName} ${lastName}`
 
 // 4 - Console log the value of `fullName`
-
+console.log(fullName);
 // 5 - Create a variable called `luckyNumber` and assign it the value of your lucky number.
-
+let luckyNumber = 7;
 // 6 - Console log this sentence, adding in the variables you created above: 
 // 'My name is (full name), and I think (lucky number) is a winner!'.
 // Refer back to the videos if you need help with this one.
-
+console.log(`My name is ${fullName}, and I think ${luckyNumber} is a winner!`);
 // 7 - Create a variable named `adventurous` and set it to a boolean value (true or false)
-
+let adventurous = true;
 // 8 - Create a variable named `food`, and set its value to a string of your favorite food
-
+let food = 'steak';
 // 9 - Create a variable called `pets` and set it to the value of the number of pets you have
-
+let pets = 1;
 // 10 - Create a variable called `friendsPets` and assign it the value of the number of pets your friend has
-
+let friendsPets = 4;
 // 11 - Add two pets to your `pets` variable
-
+pets = pets + 2;
 // 12 - Create a constant variable called `allowedPets` and set it to a number value of your choice
-
+const allowedPets = 9;
 // 13 - Create a conditional: if adventurous is true, console log "Adventures are great!", 
 // if it's not true, console log "How about we stay home?"
-
+if (adventurous === true) {
```

Comment:
Since `adventurous` is already a boolean, you could have simplified this by dropping the `=== true`:

```js
if (adventurous === true) {
    console.log('Adventures are great!');
}
```